libs/blocks/global-footer/global-footer.css

```diff
@@ -0,0 +1,399 @@
+footer {
+  font-size: var(--type-body-xxs-size);
+  padding: 1rem 0;
```

Comment:
At the moment the footer is a 1:1 copy/paste of the current footer used in Milo, https://github.com/adobecom/milo/blob/main/libs/blocks/footer/footer.js.

We're currently working on refactoring the footer and it will come in another PR. For the time being, we just wanted the Milo footer to work with our refactored Gnav.

So I will mark the footer-related comments as resolved, since this is already in production and we're re-writing it as we speak.